ios/browser/api/brave_shields/adblock_filter_list_catalog_entry.mm

```diff
@@ -15,9 +15,13 @@ @interface AdblockFilterListCatalogEntry ()
 @property(nonatomic, copy) NSString* title;
 @property(nonatomic, copy) NSArray<NSString*>* languages;
 @property(nonatomic, copy) NSString* supportURL;
+@property(nonatomic, copy) NSString* desc;
+@property(nonatomic, copy) bool hidden;
+@property(nonatomic, copy) bool default_enabled;
+@property(nonatomic, copy) bool first_party_protections;
+@property(nonatomic, copy) uint8_t permission_mask;
```

Comment:
1. `copy` doesn't apply to primative types, it can be removed entirely and just be `@property(nonatomic)`
2. You didn't expose any of these properties in the header file so they aren't accessible on the iOS side

Reply:
3. Camel-case for Obj-C property names and prefixed with `is` for bools (i.e. `isHidden`, `isEnabledByDefault`, isFirstPartyProtection`, etc)

Reply:
ah, exactly what I needed to know :pray: 

updated